syfertext/data/iterators/bert_loader.py

```diff
@@ -0,0 +1,83 @@
+from typing import Dict, List
+from torch import LongTensor
+from transformers import DataCollatorForLanguageModeling
+
+
+class BERTIterator:
+    
+    def __init__(self, dataset_reader, batch_size: int, sentence_len: int):
+        self.dataset_reader = dataset_reader
+        self.batch_size = batch_size
+        self.sentence_len = sentence_len
+
+        self.data_collator = DataCollatorForLanguageModeling(
+            tokenizer=self.dataset_reader.encoder.tokenizer_ref,
+            mlm = True,
+            mlm_probability = 0.15)
+
+    def load(self, dataset_meta) -> LongTensor:
+        self.dataset_reader.read(dataset_meta)
+
+        #In case user wants to display the data
+        return self.dataset_reader.encoded_text
```

Comment:
Please remove this, if we want to display that data, we can access the attributes of the `self.dataset_reader` object. In privacy-preserving settings, displaying the data is not desired